tests/041.solrobject_illegal_operation.phpt

```diff
@@ -48,9 +48,7 @@ Exception 1006: SolrObject instances are read-only. Properties cannot be added,
 
 Warning: main(): Attempting to set value for [newprop2_dimension_access] property in a SolrObject instance in %s on line %d
 Exception 1006: SolrObject instances are read-only. Properties cannot be added, updated or removed.
-
-Warning: main(): Attempting to remove [responseHeader] property in a SolrObject instance in %s on line %d
```

Comment:
Don't know why need, unrealted to PHP 8.
See some failure about this in travis, not always...